.github/workflows/ci-tests.yml

```diff
@@ -10,7 +10,13 @@ jobs:
   build:
    runs-on: ubuntu-latest
    env:
-     CI: true
+     APP_ENV: production
+     NODE_ENV: production
+     PANOPTES_ENV: production
+     CONTENTFUL_ACCESS_TOKEN: ${{ secrets.CONTENTFUL_ACCESS_TOKEN }}
+     CONTENTFUL_SPACE_ID: ${{ secrets.CONTENTFUL_SPACE_ID }}
+     CONTENT_ASSET_PREFIX: 'https://fe-content-pages.zooniverse.org'
+     PROJECT_ASSET_PREFIX: 'https://fe-project.zooniverse.org'
```

Comment:
Could you add the missing environment variables and their definitions to https://github.com/zooniverse/front-end-monorepo/blob/master/README.md#environment-variables

Reply:
Sure. I'll update this PR.